sycl/source/detail/scheduler/commands.cpp

```diff
@@ -245,6 +246,37 @@ cl_int MemCpyCommandHost::enqueueImp() {
   return CL_SUCCESS;
 }
 
+// SYCL has a parallel_for_work_group variant where the only NDRange
+// characteristics set by a user is the number of work groups. This does not map
+// to the OpenCL clEnqueueNDRangeAPI, which requires global work size to be set
```

Comment:
I think it's better to refer to the SYCL specification (from 4.8.5.3):
> The member function handler::parallel_for_work_group is parameterized by the number of work-groups, such that
_**the size of each group is chosen by the runtime**_, or by the number of work-groups and number of work-items for
users who need more control.

Reply:
I think my explanation gives a clearer explanation of the motives of this function and what it does. I will add this spec reference as additional info.